pkg/render/fluentd.go

```diff
@@ -144,7 +208,9 @@ func (c *fluentdComponent) Objects() ([]runtime.Object, []runtime.Object) {
 		objs = append(objs, c.filtersConfigMap())
 	}
 	if c.eksConfig != nil {
-		if c.installation.KubernetesProvider != operatorv1.ProviderOpenShift {
+		// Windows PSP does not support allowedHostPaths yet.
```

Comment:
Sorry, this one isn't just PSP right? This is for EKS specifically or am I misunderstanding?

Reply:
nvm I confused myself.